src/agenda/style.js

```diff
@@ -37,6 +37,7 @@ export default function styleConstructor(theme = {}) {
       textAlign: 'center',
       fontSize: 13,
       color: appStyle.textSectionTitleColor,
+      fontFamily: appStyle.textDayHeaderFontFamily
```

Comment:
You can add here support also for 'textDayHeaderFontSize' and 'textDayHeaderFontWeight'.